docker/docker-compose.base.yml

```diff
@@ -9,7 +9,8 @@ services:
       - discovery.type=single-node
       - plugins.security.disabled=false
       - plugins.ml_commons.model_access_control_enabled=true
-      - plugins.ml_commons.only_run_on_ml_node=false
+      - plugins.ml_commons.only_run_on_ml_node=true
```

Comment:
I still have to test, but won't this prevent the ml plugin from running on the single node ? And `true` is the default value, so no need to set it explicitly I believe.

Reply:
I will make a PR to make a cluster deployment instead of a single node deployment, so this won't be a problem once that is in.